From ddbdfd989acbd2628bc26514a8ba33cbc210e5cf Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Mon, 7 May 2018 11:15:36 +0800
Subject: [PATCH 3746/5242] MLK-18207 gpu: imx: framegen: Remove redundant pll
 and display clk rate get

commit  655b04509103353de9e7fd1f3191407dc98026e9 from
https://source.codeaurora.org/external/imx/linux-imx.git

We get pll and display clock rates twice in framegen_cfg_videomode().
This patch removes the redundant code so that the rates are got once.

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/imx/dpu/dpu-framegen.c |    3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/gpu/imx/dpu/dpu-framegen.c b/drivers/gpu/imx/dpu/dpu-framegen.c
index 3e76559..50401fc 100644
--- a/drivers/gpu/imx/dpu/dpu-framegen.c
+++ b/drivers/gpu/imx/dpu/dpu-framegen.c
@@ -267,9 +267,6 @@ void framegen_cfg_videomode(struct dpu_framegen *fg, struct drm_display_mode *m)
 	dpu_fg_write(fg, 0, FGCCR);
 	mutex_unlock(&fg->mutex);
 
-	clk_get_rate(fg->clk_pll);
-	clk_get_rate(fg->clk_disp);
-
 	disp_clock_rate = m->clock * 1000;
 
 	/* find an even divisor for PLL */
-- 
1.7.9.5

